boardwalk/prod.yml

```diff
@@ -37,7 +40,7 @@ services:
       - login-db
     restart: always
   dcc-dashboard:
-    image: quay.io/ucsc_cgl/commons-dashboard:commons-0.1.3-beta
+    image: quay.io/ucsc_cgl/commons-dashboard:commons-0.1.5-beta
```

Comment:
This should not be showing up as change. Can you please resolve it so it doesn't.

Reply:
We decided not to worry about this as the branch merges with `feature/commons`.